ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/router-bgp-base.md

```diff
@@ -224,6 +224,9 @@ router bgp 65101
    redistribute bgp leaked route-map RM-REDISTRIBUTE-BGP
    redistribute connected rcf Router_BGP_Connected()
    redistribute ospf include leaked
+   redistribute ospf match internal
```

Comment:
I am missing an example with route-map, one with route-map + include leaked. And then verify the look the same when applied on EOS. Thanks

Reply:
Added the example.